<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1116</width>
    <height>992</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1116</width>
    <height>992</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 255, 0);</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QLabel" name="txtTitle">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>50</y>
     <width>691</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>32</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 100, 0);</string>
   </property>
   <property name="text">
    <string>Goon Squad Encrypting</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSelectFile">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>230</y>
     <width>280</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton#btnSelectFile::hover
{
border: 6px solid rgb(255,255,255);
}

QPushButton#btnSelectFile{


background-color: rgb(0, 255, 0);
border: 6px solid rgb(0,100,0);

}</string>
   </property>
   <property name="text">
    <string>Choose Plain Text File</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSelectOutputDestination">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>480</y>
     <width>280</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton#btnSelectOutputDestination::hover
{
border: 6px solid rgb(255,255,255);
}

QPushButton#btnSelectOutputDestination{


background-color: rgb(0, 255, 0);
border: 6px solid rgb(0,100,0);

}</string>
   </property>
   <property name="text">
    <string>Choose Output Destination</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="taPlainText">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>200</y>
     <width>551</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255,255,255)</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="taCipherText">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>500</y>
     <width>551</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnEncrypt">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>730</y>
     <width>180</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton#btnEncrypt::hover
{
border: 6px solid rgb(255,255,255);
}

QPushButton#btnEncrypt{


background-color: rgb(0, 255, 0);
border: 6px solid rgb(0,100,0);

}</string>
   </property>
   <property name="text">
    <string>Encrypt</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDecrypt">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>730</y>
     <width>180</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton#btnDecrypt::hover
{
border: 6px solid rgb(255,255,255);
}

QPushButton#btnDecrypt{


background-color: rgb(0, 255, 0);
border: 6px solid rgb(0,100,0);

}</string>
   </property>
   <property name="text">
    <string>Decrypt</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbEncryptionTechniques">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>700</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>Ceaser Cipher</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Columnar Transposition Cipher</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="txtEncryptionTechnique">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>660</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Encryption Technique:</string>
   </property>
  </widget>
  <widget class="QLabel" name="txtInputText">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>150</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>Input Text:</string>
   </property>
  </widget>
  <widget class="QLabel" name="txtOutputText">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>440</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Output Text:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tfKey">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>790</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="txtKey">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>750</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Key:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="txtFileChosen">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>330</y>
     <width>401</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="txtOutputDestination">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>580</y>
     <width>391</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="txtAdvanced">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>820</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
QLabel#txtAdvanced::hover
{
color : rgb(52, 37, 255)
}

QLabel#txtAdvanced{


color:rgb(0,0,0);

}</string>
   </property>
   <property name="text">
    <string>Advanced</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chkBlockChain">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>910</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox::indicator {
     width: 20px;
     height: 20px;
}</string>
   </property>
   <property name="text">
    <string>Use Chaining?</string>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="taSignature">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>890</y>
     <width>191</width>
     <height>87</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="plainText">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Signature must be SHA256 in hexadecimal form</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>910</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Signature:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
